src/client/components/Headline/Headline.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react'
+import styled, { StyledComponent } from '@emotion/styled'
+import { fonts, colorsV3 } from '@hedviginsurance/brand'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+
+const { FAVORIT } = fonts
+const { gray900 } = colorsV3
+
+export type Props = {
+  variant: 'xl' | 'l' | 'm' | 's' | 'xs' | 'overline'
+  children: React.ReactNode
+}
+
+const HeadlineXL = styled.h2`
```

Comment:
🤔 I was thinking in maybe limit `as` prop value to available headings (h1 - h6) and assign `h1` as the default value. I guess that to turn a `span` into a heading we'd need to provide aria attributes to it (if we care about acessibility).

Reply:
Yup, I think we could set a default heading element, could be a `h2` or a `h3` since they are probably more often used. We could also make the as prop required if we want to enforce you to think on what heading level you're going to use.

Reply:
I think we should probably enforce it, and not have `h1` as default since that makes the risk high that we'll end up with several `h1` on the same page? But OK, I don't know enough about this `as` prop - I think that made the component render as that element? I.e. no need for aria attributes?

Reply:
Precisely, it changes what element it's rendered, so as you say no need for aria attributes :) Makes sense to not have it as an `h1` 👍